src/components/AddBookForm.jsx

```diff
@@ -11,7 +11,14 @@ const AddBookForm = () => {
     const id = uuidv4();
     const title = document.querySelector('.title-input').value;
     const author = document.querySelector('.author-input').value;
-    dispatch(addBook(id, title, author));
+    axios.post('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/OoMg7JW7xItdmWuHqq1t/books/', {
+      item_id: id,
+      title,
+      author,
+      category: 'Fiction',
+    }).then(() => {
+      dispatch(addBook(id, title, author));
+    });
```

Comment:
- **[OPTIONAL]** I think it would have been better to keep this API functionality for adding book in the books redux file to make this file strictly a component that only contains component logic. It would also mean you won't have the API URL everywhere in the project.

Reply:
Hey @DammyShittu, Thank you for your feedback. I've made the changes you suggested.